docs/faq.md

```diff
@@ -158,3 +158,10 @@ fallback profiles, or if you use specify a profile in the workspace.
 
 Copying the profile to `~/Library/MobileDevice/Provisioning Profiles` will
 resolve the error.
+
+## Why I do not see any Simulators after generating project using rules_xcodeproj?
```

Comment:
```suggestion
## Why I do not see any simulators after generating project using rules_xcodeproj?
```

Reply:
Can you move this section above `Why do I get an error like “Provisioning profile "PROFILE_NAME" is Xcode managed, but signing settings require a manually managed profile”?`

Reply:
Done